paradise_papers_django/search/static/css/base.css

```diff
@@ -182,3 +181,112 @@ option {
 .side-serch {
     width: 30%;
 }
+
+.node header {
+    margin: 2em 0;
+    position: relative;
+}
+.node header h3 {
+    margin: 0;
+    text-transform: uppercase;
+    font-size: .75em;
+    color: #999;
+}
+
+ul.relationships  {
+    font-size: 1.25em;
+    margin: 0 0 1em 0;
+}
+
+ul.relationships  li {
+    margin: 0 0 .5em 0;
+
+}
+
+ul li  {
+    padding: 0 2em 0 0;
+    list-style-type: none;
+    margin: 0 0 .5em 0;
+}
+ul li span{
+    display: inline-block;
+}
+
+.node a {
+    -webkit-transition: all 0.15s ease-out 0s;
+    -moz-transition: all 0.15s ease-out 0s;
+    transition: all 0.15s ease-out 0s;
+    color: #F8B700;
+    outline: none;
+    margin: 0 0 .5em 0;
+}
+
+.left-side{
+    width: 50%;
+}
+.right-side {
+    width: 50%;
+    border-left: 1px solid #efefef;
+    padding: 0 0 0 2em;
+    box-sizing: border-box;
+}
+
+.pure-g {
+    font-size: .85em;
+    text-rendering: optimizespeed;
+    font-family: "Lato", "Source Sans Pro", sans-serif, Arial;
+    display: -webkit-flex;
+    -webkit-flex-flow: row wrap;
+    display: -ms-flexbox;
+    -ms-flex-flow: row wrap;
+    -ms-align-content: flex-start;
+    -webkit-align-content: flex-start;
+    align-content: flex-start;
+}
+
+.search_content {
+    font-family: "Lato", "Source Sans Pro", sans-serif, Arial;
+}
+.search_results_cont.connections h2 {
+    font-size: 1em;
+    text-transform: uppercase;
+    font-weight: 700;
+    color: #852308;
+}
+
+.search_results_cont.connections h3 {
+    color: #852308;
+    font-size: 1.8em;
+    font-weight: 200;
+    margin: 0.5em 0;
+}
+
+table {
```

Comment:
Can we use a class here?

Reply:
On the h2, h3 cases, I think is specific enough that we do not need it. The table will do